pubsub/query/parser_test.go

```diff
@@ -83,9 +83,9 @@ func TestParser(t *testing.T) {
 	for _, c := range cases {
 		_, err := query.New(c.query)
 		if c.valid {
-			assert.NoError(t, err, "Query was '%s'", c.query)
+			assert.NoError(t, err, "Query was '"+c.query+"'")
```

Comment:
can't we use `assert.NoErrorf`?

Reply:
https://github.com/tendermint/tmlibs/issues/92